BUILD.md

```diff
@@ -32,6 +32,9 @@ The [Makefile](Makefile) provides a set of targets to help simplify the build
 tasks. To ensure cross-platform consistency, many of these targets use Docker
 to spawn build containers based on the `ghcr.io/kanisterio/build` public image.
 
+By default, helm looks for a file named config in the $HOME/.kube directory. You can specify other kubeconfig files by setting the KUBECONFIG environment variable or by setting the --kubeconfig flag.
+Make sure the Kuberntes cluster config is present in $HOME/.kube directory for next comamnds to successfully execute.
+
```

Comment:
Please wrap each line to 80 characters, per the rest of the documentation.

> By default, helm looks for a file

`make test` isn't using Helm. I think all we need to say here is that for `make test` to succeed, a valid kubeconfig file must be found at `$HOME/.kube/config`. See the Docker command that runs `make test` [here](https://github.com/kanisterio/kanister/blob/fa04d77eb6f5c92521d1413ddded385168f39f42/Makefile#L219).